commands/system/help.js

```diff
@@ -0,0 +1,326 @@
+const Command = require(`${process.cwd()}/base/Command.js`);
+const { MessageEmbed } = require("discord.js");
+
+const perpage = 10;
+let on = false;
+
+class Help extends Command {
+  constructor(client) {
+    super(client, {
+      name: "help",
+      description: "Get help on a command, command category, or a setting",
+      extended: "This command will display all available commands for your permission level, with the additonal option of getting per command information when you run 'help <command name>'.",
+      category: "System",
+      usage: "help <category/command/setting>",
+      aliases: ["h", "halp", "commands"],
+    });
+  }
+
+  async run(message, [type], level) {
+    var page = 1;
```

Comment:
Please do not use `var`. [no-var](https://eslint.org/docs/rules/no-var).

Reply:
I had redefining issues

Reply:
use let instead